### DISCUSSION TITLE
Python client does not create offer with bundling

### DISCUSSION THREAD

**Discussion by marc-evans** [aiortc/aiortc#754] [Q&A]
Hi,
I'm trying to create an `aiortc` client that can interact with the `pion` library. The setup is that the Python client (i.e. the "controlling" ICE agent) adds audio and video transceivers and then sends the generated offer to the `pion` client (i.e. the "controlled" ICE agent). However, the issue is I'm having is that the `pion` client throws an error:
```
SetRemoteDescription called with multiple conflicting ice-ufrag values
```
Inspecting the offer, I indeed see different ICE credentials for each media stream. So, is there a straightforward way to ensure the offer being sent from `aiortc` uses SDP bundling across media types?

**Comment by tbarbugli** [comment]
same here, did you find a way to solve this?

**Comment by fippo** [comment]
Fixed in https://github.com/aiortc/aiortc/pull/1229 (and also in pion)
